src/components/status/status.scss

```diff
@@ -0,0 +1,62 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-status-gap: var(--sbb-spacing-fixed-1x);
+  --sbb-status-icon-color: var(--sbb-color-iron-default);
+  --sbb-status-icon-margin-block: calc(
+    (
+        (var(--sbb-font-size-text-s) * var(--sbb-typo-line-height-body-text)) - var(
+            --sbb-size-icon-ui-small
+          )
+      ) / 2
+  );
```

Comment:
The complicated part about the logic could maybe directly be used where applied and only the font size could either be text-s or title-5 in a var.

Reply:
done